rcl/include/rcl/timer.h

```diff
@@ -252,7 +252,6 @@ rcl_timer_call(rcl_timer_t * timer);
  * \param[out] clock the rcl_clock_t * in which the clock is stored
  * \return #RCL_RET_OK if the period was retrieved successfully, or
  * \return #RCL_RET_INVALID_ARGUMENT if any arguments are invalid, or
- * \return #RCL_RET_TIMER_INVALID if the timer is invalid, or
  * \return #RCL_RET_ERROR an unspecified error occur.
```

Comment:
Actually, this whole bit needs to be slightly redone.  `rcl_timer_clock` *can* returned RCL_RET_TIMER_INVALID, but it can't return RCL_RET_ERROR.  Also some of the language is slightly wrong (clearly copied from elsewhere).  I'll suggest replacing this whole block with:

```
 * \return #RCL_RET_OK if the clock was retrieved successfully, or
 * \return #RCL_RET_INVALID_ARGUMENT if any arguments are invalid, or
 * \return #RCL_RET_TIMER_INVALID if the timer is invalid.
```